data/categories.txt

```diff
@@ -4763,6 +4763,318 @@ vi:Nhà vật lý trị liệu
+fa:شنوایی شناس
+fi:Audiologi
+fr:Audiologiste
+he:אודיולוג
+hu:Audiológus
+id:Audiologis
+it:Audiologo
+ja:オーディオロジスト
+ko:청력학자
+mr:ऑडिओलॉजिस्ट
+nb:Audiolog
+nl:Audioloog
+pl:Audiolog
+pt:Audiologista
+pt-BR:Fonoaudiólogo
+ro:Audiolog
+ru:Аудиолог
+sk:Audiológ
+sv:Audiolog
+sw:Mtaalamu wa kusikia
+th:นักโสตสัมผัสวิทยา
+tr:Odyolog
+uk:Аудіолог
+vi:chuyên gia thính học
+zh-Hans:听力学家
+zh-Hant:聽力學家
+
+healthcare-blood_donation
+en:Blood donation
+ar:تبرع بالدم
+be:Донарства крыві
+bg:Даряване на кръв
+cs:Darování krve
+da:Bloddonation
+de:Blutspende
+el:Αιμοδοσία
+es:Donación de sangre
+et:Veredoonorlus
+eu:Odol ematea
+fa:اهدای خون
+fi:Verenluovutus
+fr:Don du sang
+he:תרומת דם
+hu:Véradás
+id:Donor darah
+it:Donazione di sangue
+ja:献血
+ko:헌혈
+mr:रक्तदान
+nb:Bloddonasjon
+nl:Bloeddonatie
+pl:Oddawanie krwi
+pt:Doação de sangue
+pt-BR:Doação de sangue
+ro:Donarea de sânge
+ru:Донорство крови
+sk:Darovanie krvi
+sv:Blodgivning
+sw:Utoaji wa damu
+th:การบริจาคเลือด
+tr:Kan Bağışı
+uk:Донорство крові
+vi:Hiến máu
+zh-Hans:献血
+zh-Hant:獻血
+
+healthcare-optometrist
+en:Optometrist
+ar:طبيب العيون
+be:Акуліст
+bg:Оптометрист
+cs:Optometrista
+da:Optometrist
+de:Augenoptiker
+el:Οπτομέτρης
+es:Optometrista
+et:Optometrist
+eu:Optometrista
+fa:اپتومتریست
+fi:Optometristi
+fr:Optométriste
+he:אופטומטריסט
+hu:Optometrista
+id:Ahli kacamata
+it:Optometrista
+ja:オプトメトリスト
+ko:검안사
+mr:ऑप्टोमेट्रिस्ट
+nb:Optometrist
+nl:Optometrist
+pl:Optometrysta
+pt:Optometrista
+pt-BR:Optometrista
+ro:Optometrist
+ru:Оптометрист
+sk:Optometrista
+sv:Optometrist
+sw:Daktari wa macho
+th:นักทัศนมาตร
+tr:Optometrist
+uk:Оптометрист
+vi:bác sĩ nhãn khoa
+zh-Hans:验光师
+zh-Hant:驗光師
+
+healthcare-podiatrist
+en:Podiatrist
+ar:طبيب الأرجل
+be:Ортопед
+bg:Подиатрист
+cs:Podiatr
+da:Fodterapeut
+de:Podologe
+el:Ποδίατρος
+es:Podólogo
+et:Jalaravi
+eu:Podologoa
+fa:متخصص پا
+fi:Jalkaterapeutti
+fr:Podologue
+he:פודיאטר
+hu:Podiatrist
+id:Ahli penyakit kaki
+it:Podologo
+ja:足病医
+ko:발 전문의
+mr:पोडियाट्रिस्ट
+nb:Fotterapeut
+nl:Podotherapeut
+pl:Podolog
+pt:Podologista
+pt-BR:Podólogo
+ro:Podiatru
+ru:Врач-ортопед
+sk:Podológ
+sv:Fotvårdare
+sw:Daktari wa miguu
+th:แพทย์โรคเท้า
+tr:Podiatrist
+uk:Ортопед
+vi:Bác sĩ chân
+zh-Hans:足科医生
+zh-Hant:足病醫生
+
+healthcare-psychotherapist
+en:Psychotherapist
+ar:معالج نفسي
+be:Псіхатэрапеўт
+bg:Психотерапевт
+cs:Psychoterapeut
+da:Psykoterapeut
+de:Psychotherapeutin
+el:Ψυχοθεραπευτής
+es:Psicoterapeuta
+et:Psühhoterapeut
+eu:Psikoterapeuta
+fa:روان درمانگر
+fi:Psykoterapeutti
+fr:Psychothérapeute
+he:פסיכותרפיסט
+hu:Pszichoterapeuta
+id:Psikoterapis
+it:Psicoterapeuta
+ja:サイコセラピスト
+ko:심리 치료사
+mr:मानसोपचारतज्ज्ञ
+nb:Psykoterapeut
+nl:Psychotherapeut
+pl:Psychoterapeuta
+pt:Psicoterapeuta
+pt-BR:Psicoterapeuta
+ro:Psihoterapeut
+ru:Психотерапевт
+sk:Psychoterapeut
+sv:Psykoterapeut
+sw:Mwanasaikolojia
+th:นักจิตบำบัด
+tr:Psikoterapist
+uk:Психотерапевт
+vi:nhà trị liệu tâm lý
+zh-Hans:心理治疗师
+zh-Hant:心理治療師
+
+healthcare-sample_collection
+en:Sample collection
```

Comment:
Google incorrectly translated "Sample" for some languages. It's hard to translate in many cases.

Reply:
Yes it is.
I also manually adjusted the German translation.
Unfortunately it can only be mastered with help of native speakers.

I'll try to check the translations manually, sometimes deepl/goolge give alternatives.

IMO the shorter the better, in mo

Reply:
Adjusted the translations in https://github.com/organicmaps/organicmaps/pull/4053/commits/364e17743570beb3392aa841183d54a371c4f5b3
They are now based on the german "Probenahme", which in many cases translated in shorter terms.